packages/react-native/Libraries/Components/Pressable/Pressable.js

```diff
@@ -368,6 +367,12 @@ function Pressable(
     focusable: focusable !== false && !disabled, // macOS]
     accessibilityValue,
     hitSlop,
+    // [macOS
+    acceptsFirstMouse: acceptsFirstMouse !== false && !disabled,
+    enableFocusRing: enableFocusRing !== false && !disabled,
+    keyDownEvents: _keyDownEvents,
```

Comment:
If `_keyDownEvents` isn't used elsewhere, I would just inline it:
```suggestion
    keyDownEvents: keyDownEvents ?? [{key: ' '}, {key: 'Enter'}],
```